src/modules/leaderboardApi.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable */
+import { setLocalStorageID, setLocalStorage } from './localStorageFunctions.js';
+
+/* --------===========  Generate Games ID ===========-------- */
+export const createGame = async (loadPage) => {
+  let id;
+  await fetch(
+    'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(
+        {
+          name: 'Nunito Game',
+        },
+      ),
+    },
+  )
+    .then((data) => data.json()).then((data) => data.result.split(' ')[3])
+    .then((data) => {
+      id = data;
+      setLocalStorageID(id);
+      loadPage();
+    });
+};
+
+/* --------===========  Post New Score ===========-------- */
+export const postScore = async (urlID, user, score) => {
+  await fetch(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${urlID}/scores`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(
+      {
+        user,
+        score,
+      },
+    ),
+  }).then((data) => data.json());
```

Comment:
- [ ] Following the project requirements, you should only use `async` and `await` with the `fetch` method to send and receive data. Please try to avoid the `Promises`.

Reply:
I just checked the requirements again but didn't see anything saying to use only async and await or that was forbidden to use Promises. I checked some other Micronauts codes for this project and they also used them. So I don't know how to proceed... Maybe this is an optional suggestion?

Reply:
It's not optional. Please check out the requirements below 👇:
<img width="521" alt="image" src="https://user-images.githubusercontent.com/77190203/180651044-bbc57cfe-787f-4dda-8a67-8fe9cfdc1065.png">